math/convertall/Portfile

```
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+PortGroup           qt5 1.0
+
+github.setup        doug-101 convertall 0.8.0 v
+github.tarball_from releases
+
+python.versions     38 39
```

```suggestion
python.default_version 39
```

Just remembered that since this is an application, it will always default to python39. `python.versions` is normally used for python modules that can be imported in different versions of python.

See https://guide.macports.org/#reference.portgroup.python.keywords